test/bdd/steps/importer.js

```diff
@@ -29,3 +29,17 @@ Then(/^the traversal from batch "(\S+)" should contain (\d+) trail[s]* and (\d+)
     }
     expect(foundVertices, `failed to find ${numberOfVertices} vertices in the trail`).to.be.equal(numberOfVertices);
 });
+
+Then(/^the last query should return same id as last import's$/, async function () {
+    expect(!!this.state.dc, 'DC node not defined. Use other step to define it.').to.be.equal(true);
+    expect(!!this.state.lastImport, 'Nothing was imported. Use other step to do it.').to.be.equal(true);
+    expect(!!this.state.lastImport.data_set_id, 'Last imports data set id seems not defined').to.be.equal(true);
```

Comment:
data_set_id is obsoleted, please check if it is correct

Reply:
Turns out the step isn't used anywhere, so I deleted it